rectype,issueid,project_owner,project_name,actor,time,text,action,title,clean_text,Test
rectype,issueid,project_owner,project_name,actor,time,text,action,title,text,N
issue_title,412,idan,oauthlib,ViktorHaag,2016-04-12 20:13:26,"1.1.0 seems to be listed as available on PyPi, but with no associated distribution available.

If this is just a timing issue with uploading the package, I apologize for adding an issue that's effectively noise.
",start issue,"1.1.0 advertised on PyPi, but no distribution appears to be available",110 seem list avail pypi associ distribut avail If time issu upload packag I apolog ad issu that effect nois,N
issue_closed,412,idan,oauthlib,thedrow,2016-05-01 07:35:24,nan,closed issue,"1.1.0 advertised on PyPi, but no distribution appears to be available",nan,N
issue_comment,412,idan,oauthlib,eofs,2016-04-13 08:09:41,"I've the same problem, the distribution for 1.1.0 is not available:

`DistributionNotFound: No distributions matching the version for oauthlib==1.1.0`

#408 
",nan,nan,ive problem distribut 110 avail 408,N
issue_comment,412,idan,oauthlib,Starefossen,2016-04-28 06:06:18,"@idan @thedrow are there anything I, or the community, can do to help with getting the lastes version published on PyPi?
",nan,nan,idan thedrow anyth I commun help get last version publish pypi,N
issue_comment,412,idan,oauthlib,thedrow,2016-04-28 06:35:03,"Travis must have failed to push the package. I'll retry and do it manually if it fails again.
",nan,nan,travi must fail push packag ill retri manual fail,Y
issue_comment,412,idan,oauthlib,thedrow,2016-04-28 06:48:43,"I'm getting HTTPError: 400 Client Error: This filename has previously been used, you should use a different version. when I try to upload the release both manually and in travis.
Any idea what is this all about?
",nan,nan,Im get httperror 400 client error thi filenam previous use use differ version I tri upload releas manual travi ani idea,Y
issue_comment,412,idan,oauthlib,thedrow,2016-04-28 06:52:52,"Ok done. I had to rename the source distribution for some reason.
@ViktorAtD2L @eofs @Starefossen Next time ping me directly with @thedrow on such issues or catch me on twitter or something.
I hope not to disregard these kind of problems in the future.
",nan,nan,Ok done I renam sourc distribut reason viktoratd2l eof starefossen next time ping directli thedrow issu catch twitter someth I hope disregard kind problem futur,N
issue_comment,412,idan,oauthlib,r4fek,2016-04-28 11:13:24,"@thedrow now it doesn't play well with Devpi.. Advertised version is 1.1.0, but package is called 1.1.0-1. Devpi is unable to install 1.1.0 then (it sees only 1.1.0-1). Maybe you could bump release version and upload it this time correctly?
",nan,nan,thedrow doesnt play well devpi advertis version 110 packag call 1101 devpi unabl instal 110 see 1101 mayb could bump releas version upload time correctli,N
issue_comment,412,idan,oauthlib,ViktorHaag,2016-04-28 12:04:06,"I believe that this is a security feature of PyPi: once a distribution is created, you can't alter its contents, so that no two package versions have the same name. However, I understood that if you registered a new package version without providing files, you could then manually attach files to the version: I may be incorrect in believing that.
",nan,nan,I believ secur featur pypi distribut creat cant alter content two packag version name howev I understood regist new packag version without provid file could manual attach file version I may incorrect believ,N
issue_comment,412,idan,oauthlib,thedrow,2016-04-28 15:21:01,"@r4fek I'll re-release tomorrow.
",nan,nan,r4fek ill rereleas tomorrow,N
issue_comment,412,idan,oauthlib,r4fek,2016-04-28 16:22:02,"Awesome! 
",nan,nan,awesom,N
issue_comment,412,idan,oauthlib,thedrow,2016-05-01 07:34:08,"Done.
Please let me know if you find any problems.
",nan,nan,done pleas let know find problem,N
issue_comment,412,idan,oauthlib,thedrow,2016-05-01 07:35:24,"I just tested it and it's installed and upgraded just fine.
",nan,nan,I test instal upgrad fine,Y
